docs/docs.go

```diff
@@ -19,6 +19,114 @@ const docTemplate = `{
     "host": "{{.Host}}",
     "basePath": "{{.BasePath}}",
     "paths": {
+        "/v1/deck/{deckId}/dislike": {
```

Comment:
Would be nice to rename it to something that would make more sense as dislike may lead to confusion.

Something like "unlike" or coupling "like_add" and "like_remove" instead would be great.